### PR TITLE
Working

### PR DESCRIPTION
**Pull request by BSGreaves**
Done!
Remind me to ask for feedback on how to get html back into the div. For instance, here I added "br" to the end of each array item so that they would break properly on the page. (When I used html tags it omitted the tag in the comment)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+var hohohoElement = document.getElementById("coloredReindeer");
+
+
+
+for (i = 0; i < reindeer.length; i++) {
+	finalReindeer.push(colors[i] + " " + reindeer[i] + "<br>");
+}
+
+finalReindeer = finalReindeer.join(" ");
+console.log(finalReindeer);
```
Comment (DevDreed) [inline main.js]: Please remove unnecessary `console.log().`

**Review by DevDreed — CHANGES_REQUESTED** [review]
Looks good just remove unneeded console statements.

**Comment by DevDreed** [comment]
Also instead of using `<br>` using an unordered list `<ul><li></li></ul>` would make it look more like the example.

**Comment by BSGreaves** [comment]
@drdreed Hey, do I need to tag you in each comment, or only once in each Repo when I open a pull request?
Can you recommend a method of getting an `<ul>` plus the `<li>` elements into the div using innerHTML? Right now I'm pushing little pieces of HTML into the array but I suspect what I'm doing is kinda sloppy.
Appreciate it!
Edit: sorry, learning how to add code to comments.

**Comment by DevDreed** [comment]
Something like this should work.

```
let string = "<ul>";
for (let x = 0; x < array.length; x++){
	string += "<li>Something in Here</li>";
}
string += "</ul>";

domElement.innerHTML = string;
```

**Review by DevDreed — APPROVED** [review]
👍

**Comment by BSGreaves** [comment]
Test

**Comment by morecallan** [comment]
@drdreed @BSGreaves - just wanted to call you guys out for one of the best big sib/little sib reviews out there. keep it up, homies.

**Comment by BSGreaves** [comment]
Hey, yeah @drdreed is an awesome buddy. Lots of great insights. Sorry if you're subtly hinting for me to close branches... I tend to make updates, close branches, and update Dustin all at once, then wait a week before doing it again.

**Pull request merged**